app/src/NativeBase/Components/List.tsx

```diff
@@ -168,7 +169,9 @@ const List: FC<ListProps> = ({
                     <Text fontSize="xs">{item.client}</Text>
                     <ColouredTag title={item.role} />
                     <Text fontSize="xs">{item.hours}</Text>
-                    <Text fontSize="xs">{item.description}</Text>
+                    <Hyperlink linkDefault={true} linkStyle={{ color: 'blue' }}>
```

Comment:
Should this be a brand colour rather than a generic blue?  (Have you checked with Mia what colour she would like these links to be?)